keras/src/export/lite_rt_exporter.py

```diff
@@ -0,0 +1,452 @@
+import tensorflow as tf
+from keras.src.export.export_utils import get_input_signature, make_tf_tensor_spec
+import tempfile
+import os
+import numpy as np
+
+# Try to import LiteRT AOT compilation if available
+try:
```

Comment:
I think the way to do this would be to add litert to [module utils](https://github.com/keras-team/keras/blob/master/keras/src/utils/module_utils.py), import that regularly above, and then just path from the toplevel object below (e.g. `litert.python.aot.aot_compile`). Would that work?